### ISSUE TITLE
Balance is not accurately reflected

### ISSUE THREAD

**Issue by ryanswart** [perlin-network/lens#57]
In the Himitsu branch, the account api is returning
 ```json
{
"public_key":"e919a3626df31b6114ec79567726e9a31c600a5d192e871de1b862412ae8e4c0",
"balance":9999999383999825315,
"gas_balance":0,"stake":0,"reward":0,"is_contract":false
}
```
but the resulting balance displayed is `9,999,999,383.999826 PERLs`
![2019-12-06-113601_1322x85_scrot](https://user-images.githubusercontent.com/33819886/70293866-a2efd900-181c-11ea-8927-f95512b73738.png)

Most probably a rounding error somewhere, but definitely problematic.

**Comment by ryanswart** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Lens balance incorrect](https://trello.com/c/nfIQksmB/77-lens-balance-incorrect)

**Comment by ryanswart** [comment]
Uh, just realized the decimalization is different, and that it's just rounding - however it is slightly odd to have 6 decimal places when there are 9, maybe we should have some form of expansion to show that there are further points?

Also after some testing, I see the smallest decimal increase when making transactions, which makes it feel like the fee is being added instead of deducted

**Comment by claudiucelfilip** [comment]
should be fixed by #56
